src/vs/editor/common/controller/cursorCommon.ts

```diff
@@ -82,7 +82,9 @@ export class CursorConfiguration {
 	public readonly autoClosingPairsOpen: CharacterMap;
 	public readonly autoClosingPairsClose: CharacterMap;
 	public readonly surroundingPairs: CharacterMap;
-	public readonly electricChars: { [key: string]: boolean; };
+
+	private readonly _languageIdentifier: LanguageIdentifier;
+	private _electricChars: { [key: string]: boolean; };
```

Comment:
Small nit-pick: To avoid a hidden class mutation at runtime, I prefer to have all members initialized in the ctor. Can you please add `this._electricChars = null;`